myapp/app/controllers/api/task_controller.rb

```diff
@@ -9,9 +9,13 @@ def create
       # TODO: check write permission
       task.board_id = params[:board_id]
       task.modified_at = Time.now
-      task.save
 
-      render json: task, include: Task.apiInclude
+      if task.valid?
+        task.save
+        render json: task, include: Task.apiInclude
+      else
+        render json: { 'error': task.errors }
+      end
```

Comment:
いまのままでも動作上問題はないですが、こんな感じの書き方の方が多いです。
```suggestion
        render json: { 'error': task.errors } unless task.valid?
        
        task.save
        render json: task, include: Task.apiInclude
```